### DISCUSSION TITLE
Annotation Data normal fixed to Model rotation

### DISCUSSION THREAD

**Discussion by sameeryel** [google/model-viewer#3915] [Q&A]
Hi @elalish  Is there a way to fix the annotation to a particular orientation so that the data normal is linked to the model rotation. 

That way the annotation also looks like it aligned with the model instead of pointing constantly towards the camera.

**Comment by elalish** [comment]
If you want a true 3D annotation, you should author it in the glTF (a simple quad will suffice). Annotations are DOM, and so not 3D at all, just faking it.

**Reply by sameeryel** [reply]
But I want it to be interactive and clickable. If there is just a way to keep the annotation pointed in a fixed direction as it stays in the data position location (like a data normal or a data target notation) it would be amazing!

**Reply by elalish** [reply]
No problem! Just use https://modelviewer.dev/docs/#entrydocs-scenegraph-methods-materialFromPoint and if the material name matches your annotation then call your function.

**Reply by sameeryel** [reply]
Thank you!
